config.default.yaml

```diff
@@ -241,8 +262,8 @@ renewable:
     resource:
       method: wind
       turbine: NREL_ReferenceTurbine_5MW_offshore
-    capacity_per_sqkm: 2
-    correction_factor: 0.8855
+    capacity_per_sqkm: 3
```

Comment:
Important change. Why is it changed? Should be noted in release notes!

Reply:
good catch! this should indeed not be changed (I was taking it over from pypsa-earth-sec, but it should be kept as in pypsa-earth)